ftw/iframeblock/browser/configure.zcml

```diff
@@ -12,7 +12,6 @@
         name="block_view"
         permission="zope2.View"
         class=".iframeblock.IFrameBlockView"
-        template="templates/iframeblock.pt"
```

Comment:
Did you remove this to stop the template being overridden easily? 🤔 (seriously I can't quite figure it)

Reply:
Like this, it is easier to override the class.
If you just want to override the class you'd have to use this if you don't define the template in the class itself. 
```xml
<configure package="ftw.iframeblock.browser">
   ...
</configure>

```

You either get simple template overriding or simple class overriding. And because IMHO it makes more sense to put the settings in the class instead of the template (because like this you can easily change settings dynamically) I went for the easy class override.

Reply:
make sense, thanks :-) 👍